anisoap/representations/radial_basis.py

```diff
@@ -136,7 +154,9 @@ def get_num_radial_functions(self):
     # by a precision matrix (inverse of covariance) and its center.
     # The current function computes the covariance matrix and the center
     # for the provided parameters as well as choice of radial basis.
-    def compute_gaussian_parameters(self, r_ij, lengths, rotation_matrix):
+    def compute_gaussian_parameters(
+        self, r_ij, lengths, rotation_matrix, radial_gaussian_width=None
+    ):
```

Comment:
Is this meant to be an edit for later?

Reply:
yes it leaked through. Let's retain the original version, and introduce these changes in the "couple sigma to n and l" commit

Reply:
Restored.